pkg/iss/iss.go

```diff
@@ -696,7 +696,7 @@ func (iss *ISS) validateSBMessage(message *isspb.SBMessage, from t.NodeID) error
 	// TODO: This lookup is extremely inefficient, computing the membership set on each message validation.
 	//       Cache the output of membershipSet() throughout the epoch.
 	if _, ok := membershipSet(iss.config.Membership)[from]; !ok {
-		return fmt.Errorf("sender of SB message not in the membership: %d", from)
+		return fmt.Errorf("sender of SB message not in the membership: %s", from)
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)